src/kw_include.sh

```diff
@@ -0,0 +1,14 @@
+# Used for file sourcing,
+# accepts as argument the path for the file to be sourced
+function include() {
+  local filename="$1"
+  local varname=${filename%.sh}
+  varname=${varname^^}_IMPORTED
+
+  if [ -n "${!varname}" ]; then
+    return 255
```

Comment:
The ideal return error here would be something like "file already included". But there seems to be nothing like that in the errno list. Maybe this isn't even an error at all and should return 0. What do you think? Commenting here since this number was originally my suggestion

Reply:
I like the idea of just returning 0. The user is only interested in a guarantee that the file has been sourced, I see no use in knowing if the file was sourced in this function call or if it was already sourced in the past, the result is the same and has no impact for the user

Reply:
@alan-barzilay I agree